App/Course App/Course App/SharedUI/DesignSystem.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  DesignSystem.swift
+//  Course App
+//
+//  Created by Patrik Urban on 19.05.2024.
+//
+
+import SwiftUI
+import UIKit
+
+enum TextType {
```

Comment:
For readable better code structure I would suggest to split this file into multiple. In each have just one type. For sure split enums and viewModifier

Reply:
splitted